docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/github.md

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_label: GitHub
+---
+
+# GitHub
+
+## Required Secrets for Github
+
+Our webhook receiver will require a Kubernetes secret. This secret is required to contain a `secret` key in its `stringData`. We will need to provide the value we assigned to the `secret` key to GitHub in a later step so keep it handy.
+
+```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: my-secret
+      namespace: my-namespace
+    stringData:
+    # Replace 'your-secret-here' with any non-empty
+    # arbitrary string data.
+    # The key here literally needs to be named 'secret'.
+      secret: your-secret-here
+```
+
+Github will sign payloads using this secret via HMAC signature. Our webhook receiver will use this secret to verify the signature.
+
+## Github Webhook Receiver Configuration
+
+Create a new project config; specifying a Github webhook receiver that
+targets the secret we created in the last step.
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: ProjectConfig
+metadata:
+  name: my-project-config
+  namespace: my-namespace
+spec:
+  webhookReceivers: 
+    - name: my-receiver
+      github:
+        secretRef:
+          name: my-secret
+```
+
+## Retrieving the Webhook URL
+
+The secret (among other things) will be used as an input to generate
+a secure URL for our newly created webhook receiver. We can obtain
+this URL using the following command:
+
+```
+    kubectl \
+        get projectconfigs \
+        my-project-config \
+        -n my-namespace \
+        -o=jsonpath='{.status.webhookReceivers[0].url}'
```

Comment:
```suggestion
        -o=jsonpath='{.status.webhookReceivers[0].url}'
```

this will work only if GitHub webhook receiver is configured. If you've defined, let's say a Bitbucket webhook receiver first, the URL fetched wouldn't work with GitHub. Should we mention this as a separate note instead?

Reply:
Although it won't interfere with copy/paste into a shell here, mind the unnecessary leading whitespace as it renders awkwardly.

Reply:
> this will work only if GitHub webhook receiver is configured. If you've defined, let's say a Bitbucket webhook receiver first, the URL fetched wouldn't work with GitHub. Should we mention this as a separate note instead?

Let's not overcomplicate things.

Let's just make the path a bit more general: `-o jsonpath='{.status.webhookReceivers}'`